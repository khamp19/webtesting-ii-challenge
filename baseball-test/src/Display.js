import React, { Component } from 'react';

const Display = (props) => {
  return(
    <div>
      <h3>Hello from Display</h3>
      <div style={{ display: 'flex', justifyContent: 'center' }}>
        <p>Strikes: {props.strikes}</p>
        <p>Balls: {props.balls}</p>
        <p>Fouls: {props.fouls}</p>
        <p>Hits: {props.hits}</p>
        <p>Outs: {props.outs}</p>
      </div>
    </div>
  )
}

export default Display;