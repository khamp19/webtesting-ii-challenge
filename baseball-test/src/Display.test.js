import React from 'react';
import ReactDOM from 'react-dom';
import Display from './Display';
import { shallow } from 'enzyme';

//before all console.log('display props', props)

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Display />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('should receive props', () => {
  const wrapper = shallow(<Display />)
  expect(wrapper).toMatchSnapshot()
});

it('should start with props of 0', () => {
  const wrapper = shallow(<Display strikes={0} balls={0} hits={0} fouls={0} outs={0} />)
  expect(wrapper).toMatchSnapshot();
});

test.todo('should show the right number for strikes');

test.todo('should show the right number for balls');

test.todo('should show the right number for fouls');

test.todo('should show the right number for outs');

test.todo('should show the right number for hits');

test.todo('should alert end of inning at 3 strikes');