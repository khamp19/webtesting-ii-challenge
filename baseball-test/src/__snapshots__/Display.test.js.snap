// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should receive props 1`] = `ShallowWrapper {}`;

exports[`should start with props of 0 1`] = `ShallowWrapper {}`;
